pkg/controller/nodeipam/ipam/cidrset/cidr_set.go

```diff
@@ -232,9 +232,11 @@ func (s *CidrSet) Release(cidr *net.IPNet) error {
 	s.Lock()
 	defer s.Unlock()
 	for i := begin; i <= end; i++ {
-		s.used.SetBit(&s.used, i, 0)
-		s.allocatedCIDRs--
-		cidrSetReleases.WithLabelValues(s.label).Inc()
+		if s.used.Bit(i) != 0 {
```

Comment:
This is really subtle - can I ask you for some comments in both of these blocks?

Reply:
Done.